### DISCUSSION TITLE
Is possible release rio doc source code to github if it is built use rio?

### DISCUSSION THREAD

**Discussion by italanchan** [rio-labs/rio#37] [General]
Is possible release rio doc source code to github if it is built use rio?

I want learn rio from it.

**Comment by mad-moo** [comment]
Hey!

Sorry for the late reply. I'm once again surprised that GitHub doesn't send email notifications when new discussions are started.

The Rio website source isn't public. Although there isn't anything sensitive on there _yet_, its poised to share a lot of code with our server-side codebase and other internals. That's why we've kept it private for now.

With that in mind, if you're curious about anything in particular I don't mind sending you some snippets. Is there anything you want to know more about?
